src/status_im/ui/screens/syncing/views.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.ui.screens.syncing.views
+  (:require [quo.react-native :as rn]
+            [status-im.ui.screens.syncing.styles :as styles]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.components.avatars.icon-avatar :as quo2]
+            [status-im.ui.screens.syncing.sheets.sync-device-notice.views :as sync-device-notice]
+            [status-im.utils.handlers :refer [>evt]]
+            [quo2.components.buttons.button :as quo2-button]))
+
+(defn render-device [device-name device-status]
+  [:<>
+   [rn/view {:style styles/device-row}
+    [quo2/icon-avatar {:size  :medium
+                       :icon  :main-icons/placeholder20
+                       :color :primary
+                       :style {:margin-vertical 10}}]
+    [rn/view {:style styles/device-column}
+     [rn/text {:style styles/device-name} device-name]
```

Comment:
quo2 components?

Reply:
hmm this was made 3 months ago, the idea was to swiftly get the UI up and running, I shall refactor :)
And definitely using quo2 text component is the right way to go here

Reply:
fixed in latest commit :)